### PR TITLE
rename explainers to subtree-visibility

### PR DESCRIPTION
**Pull request by vmpstr**
Please take a look.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,25 +24,37 @@ The following use-cases motivate this work:
...
+* [`subtree-visibility` spec draft](https://wicg.github.io/display-locking/index.html)
+* [`contain-intrinsic-size` explainer](https://github.com/WICG/display-locking/blob/master/explainer-contain-intrinsic-size.md)
+* [`beforematch` explainer](https://github.com/WICG/display-locking/blob/master/explainer-beforematch.md)
+
 ## Quick summary
 
-In the below, "invisible to rendering/hit testing" means not drawn or returned from any hit-testing algorithms. "not drawn" is in the same sense as `visibility: hidden`, where the content still conceptually has layout sizes, but is invisible to the user.
+In the below, "invisible to rendering/hit testing" means not drawn or returned
+from any hit-testing algorithms. "not drawn" is in the same sense as
+`visibility: hidden`, where the content still conceptually has layout sizes, but
```
Comment (chrishtr) [inline README.md]: Turns out visibility: hidden also hides from elementFromPoint and other hit testing algorithms. So we can say that "invisible to rendering/hit testing" means the same as it does for "visibility: hidden" here and in the spec.

Comment (chrishtr) [reply README.md]: Same goes for the "layout sizes" comment.

File: README.md [hunk 2]
```diff
@@ -118,29 +130,29 @@ Whether or not this is the final proposed set of features is yet undecided.
 
 Three new features are proposed:
 
-1. A new `render-subtree` CSS property (spec draft to be added soon).
+1. A new `subtree-visibility` CSS property (spec draft to be added soon).
```
Comment (chrishtr) [inline README.md]:
```suggestion
1. A new `subtree-visibility` CSS property.
```

and link to spec

File: README.md [hunk 3]
```diff
@@ -118,29 +130,29 @@ Whether or not this is the final proposed set of features is yet undecided.
...
   This property controls whether DOM subtrees affected by the property are are are
-  not "invisible to rendering/hit testing" (in the same sense as `visibility: hidden`,
-  and is the mechanism by which rendering
-  work can be avoided. Some values of `render-subtree` allow the user-agent to
-  automatically manage whether subtrees affected are rendered or not. Other values
-  give the developer complete control of subtree rendering. Note that the names of the
+  not "invisible to painting/hit testing" (in the same sense as `visibility:
```
Comment (chrishtr) [inline README.md]: Remove rendundant commentary here given above definitions

**Comment by vmpstr** [comment]
PTAL

**Pull request merged**